lib/babel.js

```diff
@@ -45,36 +46,43 @@ function createRedirects(aliases, initial) {
 	)
 }
 
+function getModulePath (moduleName) {
+	return root.resolve("node_modules", moduleName)
+}
+
 /**
  * Create a babel configuration
- *
- * @param {{aliases: Dictionary}} aliases aliases needed for this conifg
  * @returns {Configuration} the config as an object, ready for use
  */
-export function createConfiguration ({aliases}) {
+export function createConfiguration ({
+	aliases,
+	modules = "auto",
```

Comment:
What does auto do?

Reply:
this is an excellent question

Reply:
`"auto"` does `"commonjs"` unless the browser supports modules.

so, we're explicitly saying `"commonjs"` instead